### ISSUE TITLE
[Error] Interface <element> not found while updating interface in Data Model.

### ISSUE THREAD

**Issue by pratik149** [directus/directus#5572]
While editing an item I realized that an Image component was replaced by a Text Input component with an image name in it. When I went to _Data Model_ to rectify the field interface. It showed me this error.
 
`Interface "<element>" not found. Reset Interface.` 

I tried editing interface of other fields, but this error is showing everywhere. And when I click on `Reset Interface` then the whole view of the Interface side menu just goes blank.

![Screenshot (97)](https://user-images.githubusercontent.com/31278436/117650133-f4d52500-b1ad-11eb-9427-d6e77fa76333.png)

---
### Project details
Directus Version: 9.0.0-rc.64
Environment: production
OS: Windows_NT 10.0.19042
Node: 14.15.5

**Comment by nickrum** [comment]
Did you run `npx directus database migrate:latest` to update the database after updating Directus to version `9.0.0-rc.64`?

**Comment by pratik149** [comment]
> Did you run `npx directus database migrate:latest` to update the database after updating Directus to version `9.0.0-rc.64`?

No, I didn't know I had to run this cmd. The issue got resolved after running this cmd though. Thanks!
